vendor/github.com/openshift/library-go/pkg/crypto/crypto.go

```diff
@@ -371,7 +371,7 @@ func (c *TLSCertificateConfig) GetPEMBytes() ([]byte, []byte, error) {
 	if err != nil {
 		return nil, nil, err
 	}
-	keyBytes, err := EncodeKey(c.Key)
+	keyBytes, err := encodeKey(c.Key)
```

Comment:
How this change happened without updating go.mod?

Reply:
good question, let me see